prometheus_client/openmetrics/parser.py

```diff
@@ -136,6 +140,8 @@ def _parse_labels(it, text):
                 raise ValueError("Invalid line: " + text)
     return labels
 
+def _validate_utf8(char):
+    return ord(char) < 65536
```

Comment:
Why is this a test for a utf-8 character?

I think you were going the right direction with the encode.

Reply:
@brian-brazil ok. Starting on line 109, how's this instead?

`utf8_str = ''.join(labelvalue)
                try:
                    utf8_str.encode('utf')
                except:
                    raise ValueError("Invalid line: " + text`

Reply:
That sounds about right.